src/workflow-testdask/testdask.py

```diff
@@ -0,0 +1,115 @@
+"""
+Example Dask workflow
+"""
+
+# pylint: disable=C0103
+
+import logging
+import ska_sdp_config
+import distributed
+
+# Initialise logging
+logging.basicConfig()
+log = logging.getLogger('main')
+log.setLevel(logging.INFO)
+
+# Obtain connection to configuration database. This is the main way to
+# both obtain information about what the workflow is supposed to do,
+# and its way to request actions from the rest of the system (e.g. ask
+# for deployments of additional software)
+config = ska_sdp_config.Config()
+
+# Find processing block configuration from the configuration. Normally
+# this workflow should only get started once one is available, and we
+# just pull the ID from an environment variable here. But because the
+# infrastructure isn't there yet, for the moment workflows are started
+# speculatively, and wait until they spot a matching processing block
+# in the configuration.
+workflow = {
+    'id': 'testdask',
+    'version': '0.0.1',
```

Comment:
While looking for a specific version looks very brittle to me, I assume that this will disappear once we have sufficient infrastructure for starting workflows.

Reply:
Exactly